jenkins/cli.sls

```diff
@@ -1,61 +1,37 @@
 {% from "jenkins/map.jinja" import jenkins with context %}
+{% import "jenkins/macros/cli_macro.jinja" as cli_macro %}
 
-{%- macro fmtarg(prefix, value)-%}
-{{ (prefix + ' ' + value) if value else '' }}
-{%- endmacro -%}
-{%- macro jenkins_cli(cmd) -%}
-{{ ' '.join(['java', '-jar', jenkins.cli_path, '-s', jenkins.master_url, fmtarg('-i', jenkins.get('privkey')), cmd]) }} {{ ' '.join(varargs) }}
-{%- endmacro -%}
+{% if grains['os_family'] == 'RedHat' %}
+  {% set listening_tool = "curl" %}
+{% else %}
+  {% set listening_tool = jenkins.netcat_pkg %}
+{% endif %}
 
-{% set timeout = 360 %}
-
-jenkins_listening:
-  pkg.installed:
-    - name: {{ jenkins.netcat_pkg }}
-  cmd.wait:
-    - name: "until nc -z localhost {{ jenkins.jenkins_port }}; do sleep 1; done"
-    - timeout: 10
-    - require:
-      - service: jenkins
-    - watch:
-      - service: jenkins
-
-jenkins_serving:
-  pkg.installed:
-    - name: curl
+check_if_jenkins_server_runs:
+  cmd.run:
```

Comment:
Also the reason why I got rid of the installation of curl is that my company has curl already installed as a default. Do you think it would be beneficial to add a sls file that installs all the dependencies for this particular jenkins-formula so that it works out of the box for everyone?